src/views/ProxyPage.vue

```diff
@@ -371,8 +371,8 @@ export default {
 }
 
 #print-content img {
-    width: 60mm;
-    height: 85mm;
+    width: 63mm;
+    height: 88mm;
```

Comment:
Called it out in your other PR, but I think your use case is just a bit different from mine.
Having a slightly smaller default size makes it easier to quickly cut cards and still have them fit in sleeves.

Could "Actual Size" maybe be a sizing option rather than the new default?

Reply:
I feel like "regular size" should refer to regular mtg card dimensions. It's what I expect when hearing "regular" when trying to print mtg cards. And if you want smaller proxies – there is "Print Scale" option. I can suggest changing:
- "Regular" size to 63mm x 88mm (100%);
- "Wimpy" to 95% (59.85mm x 83.6mm);
- "Jacked" to 105% (66.15mm x 92.4mm)
- Setting default scale to "small" (if you feel that this should be recommented configuration).